tests/test_moxunit_runtests.m

```diff
@@ -0,0 +1,228 @@
+function test_suite=test_moxunit_runtests
+    initTestSuite;
+
+function test_moxunit_runtests_basics
```

Comment:
Not as is. Its the sort of thing that you would want to use a test factory for. For example [py.test can generate parametrized tests](https://pytest.org/latest/writing_plugins.html#_pytest.hookspec.pytest_generate_tests).

The end result of with a test factory would be a lot nicer as you would only have to write the code once but there would be many (potentially combinatorially) cases tested individually and you can see which of them fail at the end instead of just that there was at least one failure in the collection of parameters.